### PR TITLE
fix: convert border color to be the same as panel dividers

### PR DESCRIPTION
**Pull request by maxmalov**
Thanks for the awesome theme, it is really nice. But one thing buzzes me every time is that borders have different colors, like the divider between tool window and editor pane is different from the default theme border color. I've made some adjustments to make them look the same.

<details>
<summary>Before</summary>

before

</details>

<details>
<summary>After</summary>

after

</details>

### CODE REVIEW

File: src/main/resources/themes/one_dark.theme.json [hunk 1]
```diff
@@ -312,8 +312,7 @@
 
       "Header": {
         "background": "#414855",
-        "inactiveBackground": "#323844",
-        "borderColor": "#21252b"
```
Comment (maxmalov) [inline src/main/resources/themes/one_dark.theme.json:316]: this borderColor setting made the header look like 2px smaller in height compared to the tabs on the left and right sides

image

File: src/main/resources/themes/one_dark.theme.json [hunk 2]
```diff
@@ -312,8 +305,7 @@
...
+        "inactiveBackground": "#21252b"
```
Comment (mskelton) [inline src/main/resources/themes/one_dark.theme.json]: Why this change?

**Review by mskelton — COMMENTED** [review]
Thanks for submitting the PR!  I'm having a little trouble seeing the differences in the screenshots so I'd like to pull your branch and look at the differences locally.

File: src/main/resources/themes/one_dark.theme.json (hunk 2 above)
Comment (maxmalov) [reply src/main/resources/themes/one_dark.theme.json]: Probably this one from tweaking around will revert it

**Comment by maxmalov** [comment]
image

Let me expand a little, I'm trying to make the highlighted borders to be the same color

File: src/main/resources/themes/one_dark.theme.json (hunk 2 above)
Comment (maxmalov) [reply src/main/resources/themes/one_dark.theme.json]: Reverted

**Review by mskelton — APPROVED** [review]
Looks good.  Thanks for contributing!

**Comment by mskelton** [comment]
@all-contributors add @maxmalov to style

**Comment by mskelton** [comment]
@all-contributors add maxmalov for design

**Comment by mskelton** [comment]
Released in version v3.3.5!
